src/data-sources/constants.ts

```diff
@@ -35,15 +35,44 @@ export const GOOGLE_SHEETS_API_SCOPES = [
  */
 export const HTTP_SOURCE_AUTH_TYPE_SELECT_OPTIONS: SelectOption< HttpAuthTypes >[] = [
 	{ label: __( 'None', 'remote-data-blocks' ), value: 'none' },
-	{ label: __( 'Bearer', 'remote-data-blocks' ), value: 'bearer' },
 	{ label: __( 'Basic', 'remote-data-blocks' ), value: 'basic' },
-	{ label: __( 'API Key', 'remote-data-blocks' ), value: 'api-key' },
+	{ label: __( 'Bearer', 'remote-data-blocks' ), value: 'bearer' },
+	{ label: __( 'Custom', 'remote-data-blocks' ), value: 'api-key' },
 ];
 export const HTTP_SOURCE_ADD_TO_SELECT_OPTIONS: SelectOption< HttpApiKeyDestination >[] = [
-	{ label: __( 'Header', 'remote-data-blocks' ), value: 'header' },
-	{ label: __( 'Query Params', 'remote-data-blocks' ), value: 'queryparams' },
+	{ label: __( 'HTTP header', 'remote-data-blocks' ), value: 'header' },
+	{ label: __( 'Query parameter', 'remote-data-blocks' ), value: 'queryparams' },
 ];
 
+/**
+ * Data source component labels
+ */
+export const HTTP_SOURCE_AUTH_VALUE_LABELS: Record< HttpAuthTypes, string > = {
+	'api-key': __( 'Value', 'remote-data-blocks' ),
+	basic: __( 'Credentials', 'remote-data-blocks' ),
+	bearer: __( 'Bearer token', 'remote-data-blocks' ),
+	none: __( 'Ignored', 'remote-data-blocks' ),
+};
+
+/**
+ * Data source component help text
+ */
+export const HTTP_SOURCE_AUTH_TYPE_HELP_TEXT: Record< HttpAuthTypes, string > = {
+	basic: __( 'The credentials will be sent in an Authorization header.', 'remote-data-blocks' ),
+	'api-key': __( 'Construct a custom HTTP header or query parameter.', 'remote-data-blocks' ),
+	bearer: __( 'The bearer token will be sent in an Authorization header.', 'remote-data-blocks' ),
```

Comment:
Minor minor thing I might be wrong about... I *think* "the Authorization header" is the more correct way to say it... but I'm not actually totally sure.

Take it or leave it.

```suggestion
	bearer: __( 'The bearer token will be sent in the Authorization header.', 'remote-data-blocks' ),
```

Mostly because there can't be more than one `Authorization` header... so it's not `an Authorization header` which implies one of multiple... maybe?

Reply:
True, there should never be more than one `Authorization` header set. But there can be zero, so I guess the question is whether using `the` implies that an `Authorization` header is already present? Let me consult an oracle.